packages/components/src/DataViewer/Text/SimpleTextKeyValue/SimpleTextKeyValue.component.js

```diff
@@ -128,11 +138,7 @@ const SimpleTextKeyValue = React.forwardRef(function SimpleTextKeyValue(
 				<span className={classNames(theme['tc-simple-text-key'], 'tc-simple-text-key')}>
 					{formattedKey}
 					{separator}
-					{displayTypes && schema && value && (
-						<span className={classNames(theme['tc-simple-text-type'], 'tc-simple-text-type')}>
-							- {schema.type.type || value.unionKey}
```

Comment:
we lose value.unionKey ?

Reply:
I added it as a workaround to handle data from pipeline preview.
I may be wrong, but I think types are defined by the schema, not the value. So I think it is relevant to only use the schema for it. `unionKey` should not be used for that I think 🤔